WordPress/WordPressTest/JetpackPluginOverlayViewModelTests.swift

```diff
@@ -1,6 +1,8 @@
 import XCTest
 @testable import WordPress
 
+
+
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/bcfb96f7b5bf2a8e02d0b5434926cab4a59d0f54/WordPress/WordPressTest/JetpackPluginOverlayViewModelTests.swift#L5"></span>Limit vertical whitespace to a single empty line; currently 3<br />
<code>vertical_whitespace</code> <code>JetpackPluginOverlayViewModelTests.swift:5</code></td>
    </tr>
  </tbody>
</table>